src/views/profile/account-view/index.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+import { IconExternalLink16 } from '@hashicorp/flight-icons/svg-react/external-link-16'
+
+import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import AuthenticatedView from 'views/authenticated-view'
+import Text from 'components/text'
+import Heading from 'components/heading'
+
+import { ProfileSidebar } from '../sidebar'
+import s from './account-view.module.css'
+import useAuthentication from 'hooks/use-authentication'
+import CopySnippet from 'components/hds-copy-snippet'
+import Card from 'components/card'
+import StandaloneLink from 'components/standalone-link'
+
+export default function ProfileAccountView() {
+	return (
+		<AuthenticatedView>
+			<SidebarSidecarLayout
+				AlternateSidebar={ProfileSidebar}
+				sidebarNavDataLevels={[]}
+			>
+				<ProfileAccountViewContent />
+			</SidebarSidecarLayout>
+		</AuthenticatedView>
+	)
+}
+
+const ProfileAccountViewContent = () => {
+	const { user } = useAuthentication()
+
+	return (
+		<div>
+			<Heading level={1} weight="bold" size={500}>
+				Account
+			</Heading>
+			<Text className={s.subheading}>
+				A read-only view of personal account info from&nbsp;
+				<a href="https://portal.cloud.hashicorp.com/account-settings">
+					HCP Account settings
+				</a>
+			</Text>
+
+			<Heading level={2} weight="bold" size={200} className={s.sectionTitle}>
+				Email
+			</Heading>
+
+			<Card className={s.section} elevation="mid">
+				<Text className={s.description}>
+					Email address associated with your profile name
+				</Text>
+				<div className={s.copySnippetWrapper}>
+					<CopySnippet textToCopy={user?.email} />
+				</div>
+			</Card>
+
+			<Heading level={2} weight="bold" size={200} className={s.sectionTitle}>
+				HashiCorp ID
+			</Heading>
+
+			<Card className={s.section} elevation="mid">
+				<Text className={s.description}>Your ID within HashiCorp</Text>
+				<div className={s.copySnippetWrapper}>
+					<CopySnippet textToCopy={user?.id} />
```

Comment:
⛏️  Nit: I don't _think_ the `?` check is necessary here, since we have `AuthenticatedView` wrapping. If it is, maybe after `const { user } = useAuthentication()` we do like `if (user === null) return null` or something to avoid the conditional checks throughout?

```suggestion
					<CopySnippet textToCopy={user.id} />
```

Reply:
removed that `?` check in https://github.com/hashicorp/dev-portal/pull/2155/commits/48c4aa04623c7a7707c06721eb61b7436e83c039